src/main/java/klerer/fallingsand/Sand.java

```diff
@@ -0,0 +1,43 @@
+package klerer.fallingsand;
+
+public class Sand {
+
+    private int[][] field = new int[3][3];
+
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+        for (int y = 0; y < 3; y++) {
+            for (int x = 0; x < 3; x++) {
+                builder.append(field[y][x]);
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**
+     * @return the value in field
+     */
+    public int get(int x, int y) {
+        return field[y][x];
+    }
+
+    /**
+     * sets the value in field to 1
+     */
+    public void put(int x, int y) {
+        field[y][x] = 1;
+    }
+
+    public void fall() {
+        // moves all sand down one square
+        if (field[1][2] == 1) {
+            field[1][3] = 1;
+        } else if (field[1][0] == 1) {
+            field[1][0] = 0;
+            field[1][1] = 1;
+        }
```

Comment:
This code might pass the tests. But if I tell you that field needs to be 100x100 and sand can appear anywhere then this isn't going to work.